lib/content-services/src/lib/search/services/search-facet-filters.service.ts

```diff
@@ -361,7 +392,7 @@ export class SearchFacetFiltersService implements OnDestroy {
     unselectFacetBucket(field: FacetField, bucket: FacetFieldBucket) {
         if (bucket) {
             bucket.checked = false;
-            this.queryBuilder.removeUserFacetBucket(field, bucket);
+            this.queryBuilder.removeUserFacetBucket(field.field, bucket);
```

Comment:
Same here, field.field does not seem correct

Reply:
Changed